### DISCUSSION TITLE
The Locations not covered by your other zones

### DISCUSSION THREAD

**Discussion by 000x2a** [mattbanks/woocommerce-local-pickup-time#194] [Q&A] [answered]
Hi,

I am not sure if this is the right place for this? Please direct me in the right direction if not. When a shipping zone is not in an accepted area, section, is there a way to remove all shipping options and put a message in to the effect of "Sorry we don't ship to this location..."? At the moment I am receiving a default of "local pickup" with no pick up times. Is this a woocommerce thing or a plugin related issue? I think "Locations not covered by your other zones" by design cannot be customised???

@timnolte

TIA

**Comment by timnolte** [comment, marked as answer]
@000x2a so first aspect is that this plugin doesn't track/manage shipping methods in terms of detecting whether a shipping method is available in a zone. This plugin only adds a Pickup Time. WooCommerce should already be notifying customers during checkout if there isn't a shipping method available.

There are 2 options available in the latest version of the plugin. One option is to make the pickup time not required, and the other option is to tie the Pickup Time to Local Shipping methods only. I would recommend making sure that you have both options checked and then also in each Zone that has a Local Shipping method and also enable the Pickup Time for each of those Local Shipping methods.
